### DISCUSSION TITLE
Next Upgrades

### DISCUSSION THREAD

**Discussion by EngEck** [raspiblitz/raspiblitz#4150] [General]
Are upgrades planned in the near future, such as on Debian Bookworm?

**Comment by rootzoll** [comment]
As soon as there is a new RaspiOS released (based on a new debian verison) we will consider an update for next (minor) release - RaspiOS releases see here: https://downloads.raspberrypi.org/raspios_arm64/images/

At the moment of writing we are rolling with `raspios_arm64-2023-05-03`

**Comment by adevoss** [comment]
PiOS Bookworm is scheduled for mid-October (together with the RPi 5)
What I read was X11 will be replaced by Wayland.

Bookworm also mentioned in #4187 b.
